src/main/java/de/tum/in/www1/artemis/service/messaging/DistributedInstanceMessageSendService.java

```diff
@@ -135,6 +135,10 @@ public void sendParticipantScoreSchedule(Long exerciseId, Long participantId, Lo
         sendMessageDelayed("participant-score-schedule", exerciseId, participantId, resultId);
     }
 
+    private void sendMessageDelayed(String destination, Long payload) {
+        exec.schedule(() -> hazelcastInstance.getTopic(destination).publish(payload), 1, TimeUnit.SECONDS);
```

Comment:
use delegation as this could lead to [shutgun surgery](https://en.wikipedia.org/wiki/Shotgun_surgery)
```suggestion
        sendMessageDelayed(destination, {payload});
```
p.s. I wrote this in github without IDE and have 0 confidence that it compiles, just to give you an idea what I mean.

Reply:
If this isn't an easy fix, then it's fine and up to you how you want to continue.

### Opinion 

I don't really understand the point about the receiving end but overall since 
I don't really understand varargs comfortably. (This is presumably true for many devs)

If varargs can be both `T` and `T[]`, then I am deeply uncomfortable with it's use overall and prefer simple arrays.

Reply:
@simon-hng [this](https://github.com/ls1intum/Artemis/blob/bugfix/distributed-send-message-types/src/main/java/de/tum/in/www1/artemis/service/messaging/InstanceMessageReceiveService.java#L135) is the receiving part behind the broker.